content/en/blog/_posts/2023-12-05-My-first-ever-KubeCon-KubeCon+CloudNativeCon-North-America-2023/index.md

```diff
@@ -0,0 +1,175 @@
+</p>
+
+Every KubeCon has more or less the same schedule except for the talks. The talks vary depending on the [CFPs](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/program/cfp/) submitted. To give a very basic plot of the event, here is a schedule of this year's KubeCon:
+
+<figure src="pic5.png" alt="KubeCon Schedule">
+
+In case you want to check the full schedule, click [here](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/program/schedule/). You can change the timezone of the schedule according to your local time.
+
+For a beginner attending KubeCon can be overwhelming. Some might consider leaving tech and open-source. But hold your nerve. There's a place for everyone in the community. As a beginner, I'd suggest carefully examining the schedule for talks aligned with your interests. Keynotes, mainly by CNCF Ambassadors, are crucial for beginners.
+
+# My experiences
+
+I want to showcase some of the talks I attended at KubeCon North America virtually.
+
+<figure src="ss1.png">
+
+**Keynote: Welcome + Opening Remarks - Priyanka Sharma, Executive Director, Cloud Native Computing Foundation**
+
+<figure src="ss2.png">
+
+**Keynote: Windy City Whirlwind: Stirring Up the Cloud Native Ecosystem - Taylor Dolezal, Head of Ecosystem, Cloud Native Computing Foundation**
+
+<figure src="ss3.png">
+
+**Keynote: Blueprint Banter: Cloud Native Conversations by the Fireside - Taylor Dolezal, Cloud Native Computing Foundation; Alolita Sharma, Apple; Mike Bowen, Blackrock; Cailyn Edwards, Shopify; Mukulika Kapas, Intuit**
+
+<figure src="ss4.png">
+
+**Humans of Cloud Native Panel – Finding Your Way in #TeamCloudNative - Mitch Connors, Aviatrix; Miranda Jaramillo, The Trevor Project; Bart Farrell, CNCF Ambassador; Whitney Lee, VMware**
+
+<figure src="ss5.png">
+
+**From Non-Tech to CNCF Ambassador: You Can Do It Too! - Julia Furst, Veeam**
+
+<figure src="ss6.png">
+
+**From Classroom to Cluster: Embracing Kubernetes as a Student and Contributing to Its Development - Nitish Kumar, Faeka Ansari & Grace Nguyen, Independent; Meha Bhalodiya, Red Hat; Leonard Pahlke, Liquid Reply**
+
+<figure src="ss7.png">
+
+**Learning Kubernetes by Chaos – Breaking a Kubernetes Cluster to Understand the Components - Ricardo Katz, VMware & Anderson Duboc, Google Cloud**
+
+<figure src="ss8.png">
+
+**[Livestream Sponsored by Vulk Coop] Cloud Native Telco Day Hosted by CNCF - Half Day Event**
+
+<figure src="ss9.png">
+
+**15,000 Minecraft Players Vs One K8s Cluster. Who Wins? - Justin Head, Super League Gaming & Cornelia Davis, Spectro Cloud**
+
+<figure src="ss10.png">
+
+**Rise of the Global Cloud Native Community: The Untold Stories - Ashutosh Kumar, Elastic & Rajas Kakodkar, VMware**
+
+<figure src="ss11.png">
+
+I attended some talks at the event, but many others were equally helpful. KubeCon offers talks on social media post-event, enabling ongoing learning. Slack channels were instrumental in global networking.
```

Comment:
```suggestion
I attended some talks at the event, but many others were equally helpful. KubeCon offers post-event social media talks, enabling ongoing learning. Slack channels were instrumental in global networking.
```

We prefer using active voice, as per the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-active-voice)